packages/netlify-cms-widget-markdown/package.json

```diff
@@ -22,6 +22,8 @@
     "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore \"**/__tests__\" --root-mode upward"
   },
   "dependencies": {
+    "dompurify": "^2.2.6",
+    "jsdom": "^16.4.0",
```

Comment:
Usage of `jsdom` breaks our build:
https://app.netlify.com/sites/cms-demo/deploys/600e81e854079d00077981c1
It is meant to be used in a Node.js environement